client/signup/steps/clone-start/index.jsx

```diff
@@ -0,0 +1,184 @@
+							fill="#E0E8ED"
+							width="20.1"
+							height="73.8"
+						/>
+						<path
+							fill="#C8D7E2"
+							d="M20.4,61.8c3.3-1.5,6.7-2.8,10.1-3.8c3.5-0.9,7-1.8,10.6-2.4c7.1-1.2,14.3-1.8,21.6-1.9 c7.2-0.1,14.4,0.2,21.6,1c7.2,0.7,14.3,1.7,21.3,2.7s14.1,1.8,21.2,2.3c7.1,0.5,14.2,0.8,21.3,1s14.2-0.1,21.2-0.9 c7.1-0.8,14.1-2.2,21.1-3.5c0.3-0.1,0.7,0.2,0.7,0.5c0.1,0.3-0.1,0.6-0.4,0.7c-6.9,2-13.9,3.6-21,4.8c-7.1,1-14.4,1.5-21.6,1.5 c-7.2,0-14.4-0.3-21.6-0.7S112.2,61.9,105,61s-14-2-21.1-2.7c-3.5-0.4-7.1-0.8-10.6-1S66.2,57,62.7,57c-7.1,0-14.2,0.4-21.2,1.2 c-7.1,0.7-14,2.2-20.8,4.3c-0.2,0-0.3-0.1-0.4-0.2C20.2,62,20.3,61.9,20.4,61.8L20.4,61.8z"
+						/>
+						<path
+							fill="#BBCDDA"
+							d="M214,178.3l-52.8,0.2h-43.8l-87.6,0.1H16.1c-1,0-2.1-0.2-3.1-0.4c-6.3-1.6-10.7-7.3-10.7-13.8v-54.8v-75H6v75 v54.8c0,4.8,3.2,9.1,7.9,10.3c0.8,0.2,1.5,0.3,2.3,0.3h13.7l87.6,0.1h43.8l51.3,0.2l0.3-33.4l0.2-17.4l0.3-17.4 c0.1-0.3,0.3-0.5,0.7-0.5c0.2,0,0.4,0.2,0.5,0.5l0.3,17.4l0.3,17.4l0.4,34.8C215.4,177.5,214.8,178.2,214,178.3L214,178.3 L214,178.3z"
+						/>
+						<path
+							fill="#004F84"
+							d="M214.8,35.5c-8.9,1.2-17.9,1.9-26.8,2.2c-8.9,0.3-17.9,0.3-26.9,0.4c-17.9,0.2-35.8,0.3-53.7,0.4 s-35.8-0.1-53.7-0.4c-9-0.1-17.9-0.1-26.9-0.4c-9-0.3-18-1-26.9-2.2V34c8.9-1.2,17.9-1.9,26.9-2.2c8.9-0.3,17.9-0.3,26.9-0.4 c17.9-0.2,35.8-0.3,53.7-0.4s35.8,0.1,53.7,0.4c8.9,0.1,17.9,0.1,26.9,0.4s18,1,26.9,2.2L214.8,35.5L214.8,35.5z"
+						/>
+					</g>
+				</svg>
+				<p className="clone-start__description">
+					{ translate(
+						'To clone your site, you will need WordPress already ' +
+							'installed on the destination site and the server ' +
+							'credentials for the destination site.'
+					) }
+				</p>
+				<Button primary className="clone-start__button" onClick={ this.goToNextStep }>
+					{ translate( 'Continue' ) }
+				</Button>
+			</Card>
+		);
+	};
+
+	render() {
+		const {
+			flowName,
+			stepName,
+			positionInFlow,
+			signupProgress,
+			originSiteName,
+			translate,
+		} = this.props;
+
+		const headerText = translate( "Let's clone %(origin)s", { args: { origin: originSiteName } } );
+		const subHeaderText = translate(
+			"You can use this to create a test or staging site, or just back up your data for safekeeping - it's up to you!"
+		);
+
+		return (
+			<StepWrapper
+				className="clone-start"
+				flowName={ flowName }
+				stepName={ stepName }
+				headerText={ headerText }
+				fallbackHeaderText={ headerText }
+				subHeaderText={ subHeaderText }
+				fallbackSubHeaderText={ subHeaderText }
+				positionInFlow={ positionInFlow }
+				signupProgress={ signupProgress }
+				stepContent={ this.renderStepContent() }
+			/>
+		);
+	}
+}
+
+export default connect( ( state, ownProps ) => {
+	const originSiteSlug = get( ownProps, 'stepSectionName', '' );
+	const site = getSiteBySlug( state, originSiteSlug );
+	const originSiteName = get( site, 'name', '' );
+
+	return {
+		originBlogId: get( site, 'ID', 0 ),
```

Comment:
Can you set the fallback here to `-Infinity`? The reason is that is easier to spot when scanning and it doesn't look like a blog id—like 0 does—so it's easier to understand that if the id is not there, it's an error.